setup.cfg

```diff
@@ -8,7 +8,7 @@ long_description = file: README.rst
 
 
 [flake8]
-ignore = E302,E701,E305,E704,F811,N811, W503
+ignore = E302,E701,E305,E704,F811,N811,W503,Y026,Y027
```

Comment:
What are these?

Reply:
The rules that lead to me breaking the code originally.
https://github.com/PyCQA/flake8-pyi/issues/284

Reply:
For the future: since flake8, unlike pylint, does not have human-readable rule identifiers, it's often hard to understand why a rule is ignored. Or what it does, even. This is why, it's a good idea to always annotate them with code comments.

Reply:
Indeed, I normally do that, but that lack of existing comments caused me to miss it.